gtfu/src/main/java/gtfu/Recipients.java

```diff
@@ -0,0 +1,33 @@
+package gtfu;
+import java.io.*;
+import java.util.*;
+
+public class Recipients {
+
+	private static final String RECIPIENTS_FILE_PATH = "src/main/resources/conf/recipients.json";
+	private Map<String, Object> recipientsMap;
+
+	public Recipients() {
+        recipientsMap = Util.parseJSONasMap(RECIPIENTS_FILE_PATH);
+	}
+
+    public String[] get(String reportType) {
+
```

Comment:
The indentation and whitespace for much of this method looks off. 

Maybe we should do the pre-commit hook formatting after all that E was talking about, to avoid this tediousness.

Reply:
Updated indentation and created https://github.com/cal-itp/graas/issues/27 for automation